<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkConfiguration</class>
 <widget class="QDialog" name="checkConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>168</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="overlapHotel">
      <property name="text">
       <string>Check Overlapping Hotel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="overlapRental">
      <property name="text">
       <string>Check Overlapping Rental</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="overlapTravel">
      <property name="text">
       <string>Check Overlapping Travel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="missingHotel">
      <property name="text">
       <string>Check Missing Hotel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
